components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -305,6 +305,9 @@ input[type="radio"] {
 li.abnormality:hover {
   background-color: #FFF9EE;
 }
+.abnormality.highlighted {
+  background-color: #FBF5EA;
+}
```

Comment:
IMO both `:hover` and `.highlighted` should have the same color. If you think the new one is better just use it for both.